Source/WebCore/css/CSSImportRule.idl

```diff
@@ -18,14 +18,18 @@
  * Boston, MA 02110-1301, USA.
  */
 
+// https://drafts.csswg.org/cssom/#cssimportrule
+
 typedef USVString CSSOMString;
 
 [
     Exposed=Window
 ] interface CSSImportRule : CSSRule {
     readonly attribute USVString href;
     [SameObject, PutForwards=mediaText] readonly attribute MediaList media;
-    [SameObject] readonly attribute CSSStyleSheet styleSheet;
+    [SameObject] readonly attribute CSSStyleSheet? styleSheet;
     readonly attribute CSSOMString? layerName;
+    // FIXME: Add 'supportsText' interface.
+    // https://bugs.webkit.org/show_bug.cgi?id=265329
```

Comment:
I would remove this FIXME, but leave a note in bug 256180 instead.

Reply:
Done. Fixed in 'd99f558' commit.